rts/lra/coordinator-quarkus/pom.xml

```diff
@@ -13,11 +13,16 @@
 
     <properties>
         <narayana.version>${project.version}</narayana.version>
+        <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
@mmusgrov would it be possible/useful to add this into the root pom.xml?

Reply:
actually I would like to move Quarkus coordinator to JDK 11 as JDK 8 support is already deprecated in Quarkus and will be removed soon - https://quarkus.io/blog/quarkus-1-4-final-released/#java-8-deprecated. This mentions dropping JDK 8 in Quarkus 1.6 (which still didn't happen up to my knowledge but we should be ready that this can come up).

Reply:
I was just commenting to get this generally applicable, but I think it should be in the jboss-parent so actually my review would be to remove this line (and https://github.com/jbosstm/narayana/blob/master/rts/pom.xml#L46) entirely - won't it come from: https://github.com/jboss/jboss-parent-pom/blob/jboss-parent-35/pom.xml#L151 - but that second remove would be cleanup for existing code so not necessary to do in this PR as I guess you can remove this line from your PR without needing to remove it from rts/pom.xml

Reply:
We follow WildFly's lead so we should only move to JDK 11 when they do.
The LRA implementation is a shim over ArjunaCore so we should use whatever that uses.
Quarkus is only one of our target platforms so that should not dictate what we do.

Reply:
I would add that JDK 8 code will run on JDK 11 so there is no urgency.
And we already test LRA in our CI against both JDKs so again there is no urgency.
What features of JDK 11 do you feel we need for LRA?